Code/Mantid/Framework/DataObjects/src/ReflectometryTransform.cpp

```diff
@@ -223,8 +229,27 @@ Mantid::API::MatrixWorkspace_sptr ReflectometryTransform::execute(
   return ws;
 }
 
+boost::optional<boost::shared_ptr<Mantid::DataObjects::TableWorkspace>>
+createTable(boost::optional<
```

Comment:
This should not be called 'createTable' because it doesn't create a table. It add's column headings to an existing table, so call it 'addColumnHeadings'